client/app/scripts/liveblog-edit/directives.js

```diff
@@ -50,40 +50,116 @@ define([
                 }
             };
         })
-        .directive('draftPosts', ['api', 'postsService',
-            function DraftPostsDirective(api, postsService) {
-                DraftPostsController.$inject = ['$scope'];
-                function DraftPostsController($scope) {
-                    var mv = this;
-                    mv.posts = [];
-                    mv.postsInfo = {};
-                    mv.selectDraftPost = $scope.onDraftPostSeleted;
-                    function updateList() {
-                        postsService.fetchDrafts(
-                            $scope.blog._id,
-                            mv.posts,
-                            mv.postsInfo
-                        );
+        .directive('lbPostsList', [
+            'postsService', 'notify', '$q',
+            function(postsService, notify, $q) {
+
+                LbPostsListCtrl.$inject = ['$scope'];
+                function LbPostsListCtrl($scope) {
+                    var vm = this;
+
+                    function fetchPage() {
+                        // stop if the limit is reached
+                        if (angular.isDefined(vm.postsMeta.total) && vm.postsMeta.total <= vm.posts.length) {
+                            return false;
+                        }
+                        // Find the next page containing new posts
+                        var page = 1;
+                        while (vm.pagination.limit * page <= vm.posts.length) {
```

Comment:
using normal pagination system for dynamical list will not work.
think if you are on page 2 and receive 3 pages of updates